.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    versioning-strategy: "increase"
```

Comment:
Seems like this will also update major versions / packages with breaking changes so we should check the dependabot PRs whether our users are affected by this breaking change so we might not update it yet

![image](https://github.com/SchwarzIT/onyx/assets/67898185/f4eae943-fd6a-457a-9470-0faf25a11fa2)
![image](https://github.com/SchwarzIT/onyx/assets/67898185/2fee9160-c48f-4d10-9b5f-391fe3d316a5)
![image](https://github.com/SchwarzIT/onyx/assets/67898185/625f67c8-2675-4974-9f65-72945ae067a1)

Reply:
Maybe we can use the ignore property for this?: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore

I am also fine if we leave it like this and check how it works in the future

Reply:
Dependabot creates PRs, so it will build and test. If we don't want an update we can configure it: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore

Reply:
But generally we SHOULD NOT ignore version udpates

Reply:
Okay lets see :)